packages/react/src/SearchField/SearchField.test.tsx

```diff
@@ -31,6 +31,14 @@ describe('Search field', () => {
     expect(component).toBeVisible()
   })
 
+  it('renders the button with a label', () => {
+    render(<SearchField.Button label="Search" />)
+
+    const component = screen.getByRole('button')
+
+    expect(component).toHaveTextContent('Search')
```

Comment:
Don't we want to use the same approach here as in the other tests? So:

```suggestion
    const component = screen.getByRole('button', { name: 'Search' })

    expect(component).toBeInTheDocument()
```